consensus/misc/eip1559/eip1559.go

```diff
@@ -132,6 +132,67 @@ func ValidateHoloceneExtraData(extra []byte) error {
 	return ValidateHolocene1559Params(extra[1:])
 }
 
+func DecodeJovian1559Params(params []byte) (uint64, uint64, uint8) {
```

Comment:
if we keep EIP1559Params as 8bytes then we might not need the encoding/decoding for this

We only then need it for extraData where it'll still be 10 bytes: 1 version byte, 8 EIP-1559 Param bytes, 1 minBaseFee byte